apache/superset
# [Explore][Adhoc Metrics] updating adhoc metric filtering

The adhoc metric control filters out all saved metrics that can be represented as simple adhoc metrics. However, at Airbnb some use cases have come up where people have wanted to use these simple saved metrics anyway so they can leverage consistent naming for these metrics across slices. As a result I have changed the filtering regex to look for metric_names like `{aggregate}__{column}`.

test plan:
- open explore view with energy_usage datasource
- verify the sum and average saved metrics do not appear
- run specs

reviewers:
@john-bodley @michellethomas @graceguo-supercat  @mistercrunch
